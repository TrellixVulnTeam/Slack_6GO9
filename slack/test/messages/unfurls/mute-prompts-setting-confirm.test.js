const MutePromptsSettingConfirm = require('../../../lib/messages/unfurls/mute-prompts-setting-confirm');

describe('MutePromptsSettingConfirm message rendering', () => {
  test('works for 24h', async () => {
    const message = new MutePromptsSettingConfirm('for 24h');
    expect(message.getAttachment()).toMatchSnapshot();
  });

  test('works indefinitely', async () => {
    const message = new MutePromptsSettingConfirm('indefinitely');
    expect(message.getAttachment()).toMatchSnapshot();
  });
});
