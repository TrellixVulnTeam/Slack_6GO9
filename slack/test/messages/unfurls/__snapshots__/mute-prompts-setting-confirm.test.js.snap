// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MutePromptsSettingConfirm message rendering works for 24h 1`] = `
Object {
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "Prompts are muted for 24h :white_check_mark:
You can adjust this setting by invoking \`/github settings\`",
}
`;

exports[`MutePromptsSettingConfirm message rendering works indefinitely 1`] = `
Object {
  "color": "#24292f",
  "mrkdwn_in": Array [
    "text",
  ],
  "text": "Prompts are muted indefinitely :white_check_mark:
You can adjust this setting by invoking \`/github settings\`",
}
`;
